### PR TITLE
fix(UI):remove deactivated users from moderators list

### PR DESCRIPTION
**Pull request by rudra-superrr**
Signed-off-by: rudra-superrr <rudra.chopra@siemens.com>

Closes: #1758 

### How To Test?
- Create Moderation Request.
- After creating, check whether deactivated users are present in moderators list or not in Requests Portlet.

Run Script:
To remove deactivated users from the moderators list from database for old data.

### CODE REVIEW

File: scripts/migrations/054_remove_inactiveUsers_from_moderators.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script is for changing the ecc status to Approved for the affected releases (releases with source code download url and component type OSS).
```
Comment (akapti) [inline scripts/migrations/054_remove_inactiveUsers_from_moderators.py:18]: Kindly change the description of the script.

**Review by akapti — CHANGES_REQUESTED** [review]
code looks good to me.
will approve the PR once testing is completed.

**Review by akapti — COMMENTED** [review]
changes looks good to me. will approve once PR is tested.

**Comment by nikkuma7** [comment]
Test successful.

**Review by akapti — APPROVED** [review]
@nikkuma7  has tested the PR, hence approving.

**Pull request merged**